x-pack/packages/security/plugin_types_common/src/authentication/index.ts

```diff
@@ -5,6 +5,10 @@
  * 2.0.
  */
 
-export type { AuthenticatedUser, UserRealm } from './authenticated_user';
-export type { User } from './user';
-export type { AuthenticationProvider } from './authentication_provider';
+// re-exporting from core package to avoid changing all imports
+export type {
+  User,
+  UserRealm,
+  AuthenticatedUser,
+  AuthenticationProvider,
+} from '@kbn/core-security-common';
```

Comment:
I re-exported the moved type from the original module (and so package) to avoid changing all the import statements all over the place. 

Please tell me if we think we have a reason to not do this.